baystation12.dme

```diff
@@ -745,6 +745,7 @@
 #include "code\game\machinery\vending.dm"
 #include "code\game\machinery\vending_deconstruction.dm"
 #include "code\game\machinery\vitals_monitor.dm"
+#include "code\game\machinery\Vox.dm"
```

Comment:
don't devs enforce "lowercase only"?

Reply:
Normally, for file names like that, yes.